### PR TITLE
Add recipe for connectholland/timchimp-bundle

### PR DESCRIPTION
**Pull request by reyostallenberg**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: connectholland/timechimp-bundle/1.0/manifest.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "bundles": {
+        "ConnectHolland\\TimechimpBundle\\ConnecthollandTimechimpBundle": ["all"]
+    },
+    "env": {
+        "TIMECHIMP_ACCESS_TOKEN": "baerer-token-value"
```
Comment (Nyholm) [inline connectholland/timechimp-bundle/1.0/manifest.json:6]: I dont see that this is used by the bundle's config.

**Comment by reyostallenberg** [comment]
It's used here https://github.com/ConnectHolland/timechimp-bundle/blob/36e6b5209e2b2dc499281c19eaa084f89ad25d35/src/Resources/config/services.yaml#L11 

The rest is auto wired

**Comment by Nyholm** [comment]
The issue with your current setup is that if a user is missing the environment variable, they will not get an exception until runtime. 
I suggest you create some configuration to your bundle and make sure to use that env var there. If the env var is missing then, the user will get an error at build time, which is much more preferred.

**Comment by reyostallenberg** [comment]
@Nyholm then this is more like you prefer, I think.

**Pull request merged**